charts/ingress-nginx/templates/controller-service.yaml

```diff
@@ -25,6 +25,9 @@ spec:
 {{- if .Values.controller.service.externalIPs }}
   externalIPs: {{ toYaml .Values.controller.service.externalIPs | nindent 4 }}
 {{- end }}
+{{- if .Values.controller.service.externalName }}
+  externalName: {{ .Values.controller.service.externalName }}
+{{- end }}
```

Comment:
As already stated in the comments, please also implement this for the `controller-service-internal.yaml`.